CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+import: fix a copule of bugs on Shearwater Cloud import
```

Comment:
you are bringing the old CHANGELOG back :-)
No worries, I'll fix this in post 🤣